src/spawn.ml

```diff
@@ -15,7 +19,11 @@ module Unix_backend = struct
   let default =
     match Sys.getenv "SPAWN_USE_FORK" with
     | _ -> Fork
-    | exception Not_found -> Vfork
+    | exception Not_found ->
+      if is_osx then
+        Fork
```

Comment:
May be worth adding some comment here dumping your knowledge so far of quirks of vfork on OSX. (if theres anything to dump, that is)

Reply:
Sounds good, I added a comment.